modules/siddhi-core/src/main/java/org/wso2/siddhi/core/ExecutionPlanRuntime.java

```diff
@@ -159,6 +166,10 @@ public InputHandler getInputHandler(String streamId) {
         return eventSourceMap.values();
     }
 
+    public ExecutionPlanContext getExecutionPlanContext() {
```

Comment:
This was required Only for the test case Suho https://github.com/nirmal070125/siddhi/blob/c75f749c4008246806ca2bb3b70590faf70379d4/modules/siddhi-extensions/input-transports/kafka-input-transport/src/test/java/org/wso2/siddhi/extension/input/transport/kafka/KafkaInputTransportTestCase.java#L262.  Will use a separate thread pool for the test case and update the PR.